[2024-12-30T22:23:27.011+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-30T22:23:27.025+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_kafka_producer.send_success_email scheduled__2024-12-30T22:23:20+00:00 [queued]>
[2024-12-30T22:23:27.031+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_kafka_producer.send_success_email scheduled__2024-12-30T22:23:20+00:00 [queued]>
[2024-12-30T22:23:27.032+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-30T22:23:27.045+0000] {taskinstance.py:2889} INFO - Executing <Task(EmailOperator): send_success_email> on 2024-12-30 22:23:20+00:00
[2024-12-30T22:23:27.052+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=332) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-30T22:23:27.053+0000] {standard_task_runner.py:72} INFO - Started process 333 to run task
[2024-12-30T22:23:27.053+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_kafka_producer', 'send_success_email', 'scheduled__2024-12-30T22:23:20+00:00', '--job-id', '1036', '--raw', '--subdir', 'DAGS_FOLDER/weather_kafka_producer.py', '--cfg-path', '/tmp/tmpxm8sxmrb']
[2024-12-30T22:23:27.055+0000] {standard_task_runner.py:105} INFO - Job 1036: Subtask send_success_email
[2024-12-30T22:23:27.091+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_kafka_producer.send_success_email scheduled__2024-12-30T22:23:20+00:00 [running]> on host 3bad290ab57a
[2024-12-30T22:23:27.164+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_kafka_producer' AIRFLOW_CTX_TASK_ID='send_success_email' AIRFLOW_CTX_EXECUTION_DATE='2024-12-30T22:23:20+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-30T22:23:20+00:00'
[2024-12-30T22:23:27.165+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-30T22:23:27.186+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/email.py:155: RemovedInAirflow3Warning: Fetching SMTP credentials from configuration variables will be deprecated in a future release. Please set credentials using a connection instead.
  send_mime_email(e_from=mail_from, e_to=recipients, mime_msg=msg, conn_id=conn_id, dryrun=dryrun)

[2024-12-30T22:23:27.187+0000] {configuration.py:1073} WARNING - section/key [smtp/smtp_user] not found in config
[2024-12-30T22:23:27.187+0000] {email.py:272} INFO - Email alerting: attempt 1
[2024-12-30T22:23:27.290+0000] {email.py:283} INFO - Sent an alert email to ['your_email@example.com']
[2024-12-30T22:23:27.303+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-30T22:23:27.304+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_kafka_producer, task_id=send_success_email, run_id=scheduled__2024-12-30T22:23:20+00:00, execution_date=20241230T222320, start_date=20241230T222327, end_date=20241230T222327
[2024-12-30T22:23:27.348+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-30T22:23:27.362+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-30T22:23:27.364+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
